src/components/ec-currency-input/ec-currency-input.spec.js

```diff
@@ -45,6 +51,61 @@ describe('EcCurrencyInput', () => {
     expect(wrapper.element).toMatchSnapshot();
   });
 
+  describe('when the bottom note is defined', () => {
+    it('should render properly', () => {
+      const wrapper = mountCurrencyInput({ bottomNote: 'Bottom note message' });
+      expect(wrapper.findByDataTest('ec-currency-input__bottom-note').element).toMatchSnapshot();
+    });
+
+    describe('and it is a warning note', () => {
+      it('should render properly', () => {
+        const wrapper = mountCurrencyInput({
+          bottomNote: 'Bottom note message',
+          isWarning: true,
+        });
+        expect(wrapper.findByDataTest('ec-currency-input__bottom-note').element).toMatchSnapshot();
+      });
+
+      describe('and there is a warning message defined', () => {
+        it('should render properly', () => {
+          const warningMessage = 'Warning message';
+          const wrapper = mountCurrencyInput({
+            bottomNote: 'Bottom note message',
+            isWarning: true,
+            warningMessage,
+          });
+          expect(wrapper.findByDataTest('ec-currency-input__bottom-note').element).toMatchSnapshot();
+          expect(wrapper.findByDataTest('ec-currency-input__warning-tooltip').attributes('mocked-tooltip-content')).toBe(warningMessage);
+        });
+
+        describe('and an error message is being displayed', () => {
+          it('should not display neither the bottom note nor the warning icon', () => {
+            const wrapper = mountCurrencyInput({
+              bottomNote: 'Bottom note message',
+              errorMessage: 'Random message',
+              isWarning: true,
+              warningMessage: 'Warning message',
```

Comment:
@jafcalvente Should we see or not the warningMessage?

Reply:
we shouldn't. The error message always has priority and when it is defined neither the bottom note nor the warning icon (which contains the tooltip) should be displayed.